__test__/services/configBranch.test.js

```diff
@@ -1,9 +1,10 @@
 // do not put these tests on the same file as config.test.js or else jest will fail coverage
 // (describe.beforeEach scope not working)
 
+const originalPlatform = process.platform;
+
```

Comment:
with the changes made in jest, it broke this functionality (using "this" in beforeEach is now undefined)

Reply:
jest is weird. Maybe add a comment here for why we need to know the platform?